public/install-go.sh

```diff
@@ -7,11 +7,10 @@
 deps=( curl jq )
 
 for dep in "${deps[@]}"; do
-    if ! command -v "$dep" &> /dev/null; then
-        echo "$dep is not installed. Downloading and executing the script..."
-        curl -sSL https://rollkit.dev/install-jq.sh | bash
-        exit $?
-    fi
+	if ! command -v "$dep" &> /dev/null; then
+		echo "$dep is not installed. Downloading and executing the script..."
+		curl -sSL https://rollkit.dev/install-jq.sh | bash
```

Comment:
Am I understanding this correctly in that if `curl` was not installed we would still run the install jq script? 
Seems like we need to clean up with for loop or just hard code the 2 dependency checks, which might be easier and cleaner.

Reply:
If curl is not installed we would try to install it using curl :)
Fixing it